### ISSUE TITLE
Missing Upcoming Race Data in meeting

### ISSUE THREAD

**Issue by WilkPrintes** [br-g/openf1#112]
The latest endpoint in meeting only returns the most recent past race, but it would be helpful if it included upcoming races as well

**Comment by chrisn-au** [comment]
I would like to extend this and ask for place holder meetings and sessions for the remainder of the year.

**Comment by chrisn-au** [comment]
Yes, quite disturbing. Looking forward to future meetings/sessions and while not wanting to pre-empt design decisions, the way I'm planning to work around this issue in the short term is to use a predictable temporary key scheme with metadata tracking.

  The approach I'm going to implement (absolutely no need for the design to follow):
  - Generate temporary keys using compressed date encoding that stays within 32-bit integer limits
  - Add an isTemporary: true flag to the document metadata (really only needed for auditing process)
  - When official F1 data becomes available with the real keys, migrate all references and remove the temporary flag

  Meeting Key Formula: 1,000,000 + (YY * 10000) + (MM * 100) + DD

  For example, Italian GP starting Sept 5, 2025 would get temporary meeting key 1,250,905 (decodes as 2025-09-05).

  Session Key Formula: (Meeting Key * 100) + Session Sequence

  Session keys for Italian GP would be:
  - Practice 1: 125,090,501
  - Practice 2: 125,090,502
  - Practice 3: 125,090,503
  - Qualifying: 125,090,504
  - Race: 125,090,505

  This gives me:
  1. Self-documenting: Key 1,250,905 clearly represents 2025-09-05
  2. Chronological: Natural date ordering preserved
  3. Collision-free: 1M+ offset ensures no overlap with official keys (currently max 1,277)
  4. Extractable: Can decode date from key and derive meeting from session mathematically

The migration becomes straightforward as all I need to know for the weekend is the day the event starts and the session order - the temporary keys can then be mapped directly to whatever official keys F1 assigns. When official keys arrive (e.g., meeting 1278 with sessions 9901-9905), I'll migrate all references and remove the temporary flags.

  Happy to elaborate on the migration strategy if this approach makes sense for the project.
